modules/webbrowser/properties/propertywidgetcef.cpp

```diff
@@ -33,6 +33,42 @@
 
 namespace inviwo {
 
+class CefDOMSearchId : public CefStringVisitor {
```

Comment:
Add overall description of what it does and why it is needed, i.e. 
Checks if widget html id exist in the frame and sets it if it does. 
@note Cannot use CefDOMVisitor since it requires the renderer process.